plasma_framework/test/src/exits/payment/PaymentExitGame.test.js

```diff
@@ -80,6 +80,7 @@ contract('PaymentExitGame - End to End Tests', ([_, richFather, bob]) => {
 
     const setupContracts = async () => {
         this.framework = await PlasmaFramework.new(MIN_EXIT_PERIOD, INITIAL_IMMUNE_VAULTS, INITIAL_IMMUNE_EXIT_GAMES);
+        await this.framework.init();
```

Comment:
Looking at this, I think the function name `init()` might be misleading.
Perhaps make the function name `initAuthority` instead?